src/test/java/tests/ConfigFileTest.java

```diff
@@ -0,0 +1,108 @@
+package tests;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+import prefs.Config;
+import prefs.ConfigFile;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+
+public class ConfigFileTest {
```

Comment:
To ensure that `ConfigFile` works with other config, we should have a test for using session config as well. It doesn't have to be broken down like Dummy Config; put some values, write to file, read the file and check if the values are correct would suffice.